x/emissions/keeper/keeper.go

```diff
@@ -3312,21 +3312,13 @@ func (k *Keeper) AddTopicFeeRevenue(ctx context.Context, topicId TopicId, amount
 
 // return the blocks per week
 // defined as the blocks per month divided by 4.345
-func (k *Keeper) calculateBlocksPerWeek(ctx context.Context) (alloraMath.Dec, error) {
-	moduleParams, err := k.GetParams(ctx)
-	if err != nil {
-		return alloraMath.Dec{}, errorsmod.Wrap(err, "error getting params")
-	}
-	blocksPerMonth, err := alloraMath.NewDecFromUint64(moduleParams.BlocksPerMonth)
+var weeksPerMonth = alloraMath.MustNewDecFromString("4.345") //nolint:gochecknoglobals // constant
+func (k *Keeper) CalculateBlocksPerWeek(ctx context.Context, blocksPerMonth uint64) (alloraMath.Dec, error) {
```

Comment:
```suggestion
func CalculateBlocksPerWeek(ctx context.Context, blocksPerMonth uint64) (alloraMath.Dec, error) {
```

If we don't interact with the keeper maybe we can detach the func from it

Reply:
Moved to `math/utils`